package.json

```diff
@@ -78,55 +78,62 @@
     "test:ci:integration": "yarn start-test start:all 8080 cy:run"
   },
   "devDependencies": {
-    "@angular-builders/custom-webpack": "12.1.2",
-    "@angular-devkit/build-angular": "12.2.7",
-    "@angular-devkit/core": "12.2.7",
-    "@angular-devkit/schematics": "12.2.7",
-    "@angular/animations": "12.2.7",
-    "@angular/cli": "12.2.7",
-    "@angular/common": "12.2.7",
-    "@angular/compiler": "12.2.7",
-    "@angular/compiler-cli": "12.2.7",
-    "@angular/core": "12.2.7",
-    "@angular/elements": "12.2.7",
-    "@angular/forms": "12.2.7",
-    "@angular/language-service": "12.2.7",
-    "@angular/platform-browser": "12.2.7",
-    "@angular/platform-browser-dynamic": "12.2.7",
-    "@angular/router": "12.2.7",
-    "@commitlint/cli": "^8.3.5",
-    "@commitlint/config-conventional": "^8.3.4",
-    "@nrwl/angular": "12.9.0",
-    "@nrwl/cli": "12.9.0",
-    "@nrwl/nx-cloud": "12.3.13",
-    "@nrwl/workspace": "12.9.0",
-    "@types/jest": "^26.0.10",
+    "@angular-builders/custom-webpack": "13.0.0",
+    "@angular-devkit/build-angular": "13.0.3",
+    "@angular-devkit/core": "13.0.3",
+    "@angular-devkit/schematics": "13.0.3",
+    "@angular/animations": "13.0.3",
+    "@angular/cli": "13.0.3",
+    "@angular/common": "13.0.3",
+    "@angular/compiler": "13.0.3",
+    "@angular/compiler-cli": "13.0.3",
+    "@angular/core": "13.0.3",
+    "@angular/elements": "13.0.3",
+    "@angular/forms": "13.0.3",
+    "@angular/language-service": "13.0.3",
+    "@angular/platform-browser": "13.0.3",
+    "@angular/platform-browser-dynamic": "13.0.3",
+    "@angular/router": "13.0.3",
+    "@commitlint/cli": "^15.0.0",
+    "@commitlint/config-conventional": "^15.0.0",
+    "@nrwl/angular": "13.2.1",
+    "@nrwl/cli": "13.2.1",
+    "@nrwl/nx-cloud": "12.5.4",
+    "@nrwl/workspace": "13.2.1",
+    "@types/jest": "27.0.3",
     "@types/karma": "^6.1.0",
-    "@types/node": "14.14.33",
+    "@types/node": "15.12.1",
+    "@types/react": "^17.0.37",
+    "@types/react-dom": "^17.0.11",
     "@types/single-spa-react": "^2.12.0",
     "@typescript-eslint/eslint-plugin": "4.19.0",
     "@typescript-eslint/parser": "4.19.0",
     "concurrently": "^5.1.0",
     "cpx": "^1.5.0",
-    "cypress": "^8.4.1",
+    "cypress": "8.7.0",
     "eslint": "7.22.0",
     "eslint-plugin-prefer-arrow": "^1.2.0",
-    "husky": "^4.2.5",
-    "jest": "^26.6.3",
+    "husky": "^7.0.4",
```

Comment:
👍🏿  thanks for properly upgrading from husky 4 by creating the .husky directory with commit-msg, etc